### PR TITLE
Deprecate apps docs

### PR DESCRIPTION
**Pull request by JV0812**
Deleting "Docker Enterprise Edition" and "AWS Elastic Load Balancing Metrics" docs.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -671,11 +671,7 @@
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic/01-Collect-logs-for-the-AWS-Elastic-Load-Balancing-App": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic/AWS-Elastic-Load-Balancer---Classic-App-Dashboards": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
-  "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_Metrics": "/docs/integrations/amazon-aws/elastic-load-balancing-metrics",
```
Comment (kimsauce) [inline cid-redirects.json:674]: @JV0812 , we don't delete cid-redirects links without redirecting to something new. Those legacy links (e.g., `/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_Metrics`) are still out there and users will get 404s for all of them.

**Review by kimsauce — CHANGES_REQUESTED** [review]
See https://github.com/SumoLogic/sumologic-documentation/pull/2559/files#r1220127354

File: cid-redirects.json (hunk 1 above)
Comment (JV0812) [reply cid-redirects.json:674]: Thanks Kim, I have updated the URL to the closest equivalent content.

File: blog-service/2016/12-31.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ To view the full archive, [click here](/release-notes-service/archive).
...
 
-**Sumo Logic App for Amazon Inspector (Preview). **Amazon Inspector allows you to monitor your AWS resources for potential security risks. The Sumo Logic App for Amazon Inspector provides preconfigured searches and Dashboards that give  you instant access to an overview of Amazon Inspector as well as details on assessments, runs, and findings. For details see[ Amazon Inspector App](/docs/integrations/amazon-aws/inspector-classic). This is a Preview App (the Preview Apps section was deleted in 2022).
+**Sumo Logic App for Amazon Inspector (Preview).** Amazon Inspector allows you to monitor your AWS resources for potential security risks. The Sumo Logic App for Amazon Inspector provides preconfigured searches and Dashboards that give  you instant access to an overview of Amazon Inspector as well as details on assessments, runs, and findings. For details see[ Amazon Inspector App](/docs/integrations/amazon-aws/inspector-classic). This is a Preview App (the Preview Apps section was deleted in 2022).
```
Comment (kimsauce) [inline blog-service/2016/12-31.md:45]:
```suggestion
**Sumo Logic App for Amazon Inspector (Preview).** Amazon Inspector allows you to monitor your AWS resources for potential security risks. The Sumo Logic App for Amazon Inspector provides preconfigured searches and Dashboards that give  you instant access to an overview of Amazon Inspector as well as details on assessments, runs, and findings. For details, see [Amazon Inspector App](/docs/integrations/amazon-aws/inspector-classic). This is a Preview App (the Preview Apps section was deleted in 2022).
```

File: cid-redirects.json (hunk 1 above)
Comment (kimsauce) [reply cid-redirects.json:674]: thank you!

Comment (kimsauce) [reply cid-redirects.json:674]: Thank you! So it looks like you've updated the legacy redirects, which is correct. However, you also need to write redirects from the deprecated doc URLs to the new doc URLs. Deprecated docs links exist elsewhere (Google search results, saved bookmarks, etc.). [Here's the instructions.](https://help.sumologic.com/docs/contributing/remove-doc/)

For example:
`"/docs/integrations/amazon-aws/elastic-load-balancing-metrics": "/docs/integrations/amazon-aws/elastic-load-balancer-app",`

**Review by kimsauce — CHANGES_REQUESTED** [review]
Redirects still needed

**Review by kimsauce — CHANGES_REQUESTED** [review]
Redirects still needed

**Review by kimsauce — APPROVED** [review]

File: cid-redirects.json (hunk 1 above)
Comment (kimsauce) [reply cid-redirects.json:677]: @JV0812 I've gone ahead and fixed these

**Pull request merged**